limacharlie/sync.go

```diff
@@ -65,11 +65,9 @@ func (r OrgSyncFPRule) DetectionEquals(fpRule FPRule) bool {
 }
 
 type OrgSyncIntegrityRule struct {
-	LastUpdated uint64   `json:"updated" yaml:"updated"`
-	CreatedBy   string   `json:"by" yaml:"by"`
```

Comment:
Why are we not keeping those?

Reply:
To make it clear clearer and idempotent when a change has occurred. Otherwise if:
- You do a push of rule A
- Export the config
- I do a push of rule A, same exact rule
- Export the config
- We end up with 2 configs that are different but that are actually the same value.